Libs/MRML/Core/vtkMRMLNodePropertyMacros.h

```diff
@@ -0,0 +1,249 @@
+/*=auto=========================================================================
```

Comment:
Macros that simplify MRML node property read/write XML, copy, and printself.
@jcfr @pieper @cpinter